app/src/main/res/values-ar/strings.xml

```diff
@@ -438,7 +438,6 @@
 <string name="compose_preview_image_description">إجراءات على الصورة %s</string>
 
     <string name="caption_notoemoji">حزمة الإيموجي الحالية لـ غوغل</string>
-    <string name="description_poll">استطلاع رأي بالخيارات: %s, %s, %s, %s; %s</string>
```

Comment:
unfortunately I do not know how these have to look correctly in Arabic 
cc @BoFFire

Reply:
I'm checking this string. Thanks.